src/views/f2f/prove-identity-post-office.html

```diff
@@ -20,28 +20,24 @@ <h2 class="govuk-heading-m" id="firstStepTitle">{{ "landingPage.firstStepTitle"
         </div>
         <p class="instruction" id="firstStepOne">{{ "landingPage.instructions.firstStep.one" | translate }}</p>
         <p class="instruction" id="firstStepTwo">{{ "landingPage.instructions.firstStep.two" | translate }}</p> 
-        <p class="instruction" id="firstStepThree">{{ "landingPage.instructions.firstStep.three" | translate }}</p>  
+        <p class="instruction" id="firstStepThree">{{ "landingPage.instructions.firstStep.three" | translate }}</p>
       </li>
-        
+
       <li>
         <div>
           <p class="circle">{{ "landingPage.secondCircle" | translate }}</p>
           <h2 class="govuk-heading-m" id="secondStepTitle">{{ "landingPage.secondStepTitle" | translate }}</h2>
         </div>
         <p class="instruction" id="secondStepOne">{{ "landingPage.instructions.secondStep.one" | translate }}<a href={{ "common.postoffice.pathToPoBranchSearch" | translate }} target="_blank"> {{ "common.postoffice.postofficeLink" | translate }}</a></p>
-        <p class="instruction" id="secondStepTwo">{{ "landingPage.instructions.secondStep.two" | translate }}</p> 
-      </li>
-      
+        <p class="instruction" id="secondStepTwo">{{ "landingPage.instructions.secondStep.two" | translate }}</p>
+
       {% set insetBody %}
-      <div>
-        <p class="instruction" id="secondStepThree">{{ "landingPage.instructions.secondStep.three" | translate }}</p>
-      </div>
+        {{- "landingPage.instructions.secondStep.three" | translate -}}
```

Comment:
looking at the ticket description, it doesn't seem <div> <p class...> is the issue